src/platform/intel/cavs/lib/pm_runtime.c

```diff
@@ -92,6 +93,22 @@ static inline void cavs_pm_runtime_en_ssp_clk_gating(uint32_t index)
 }
 #endif
 
+static inline void cavs_pm_runtime_en_ssp_power(uint32_t index)
```

Comment:
nit-pick: I never realized we had an _en_ and _dis_ versions of the functions. Could we look at using a boolean parameter instead? IMHO it's more intuitive

Reply:
I don't like en/dis naming also, but it's not a subject of this PR, I'd like to keep it related to things that are needed for TGL.